modules/os2forms_fasit/src/Helper/Settings.php

```diff
@@ -2,115 +2,109 @@
 
 namespace Drupal\os2forms_fasit\Helper;
 
-use Drupal\Core\KeyValueStore\KeyValueFactoryInterface;
-use Drupal\Core\KeyValueStore\KeyValueStoreInterface;
-use Drupal\os2forms_fasit\Exception\InvalidSettingException;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ImmutableConfig;
+use Drupal\key\KeyRepositoryInterface;
```

Comment:
We should note somewhere, e.g. in the changelog, that we're making this change (in a major release).

Reply:
Added note in fa892f9746d714a6f73099684fb9e86d5180ef69.